extensions/amp-youtube/0.1/test/test-amp-youtube.js

```diff
@@ -17,15 +17,30 @@
 import {createIframePromise} from '../../../../testing/iframe';
 require('../amp-youtube');
 import {adopt} from '../../../../src/runtime';
+import {timer} from '../../../../src/timer';
+import * as sinon from 'sinon';
 
 adopt(window);
 
 describe('amp-youtube', () => {
+  let sandbox;
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+    sandbox = null;
+  });
 
   function getYt(videoId, opt_responsive, opt_beforeLayoutCallback) {
     return createIframePromise(
         true, opt_beforeLayoutCallback).then(iframe => {
           const yt = iframe.doc.createElement('amp-youtube');
+          sandbox.stub(yt.implementation_, "getPlayerReadyPromise_", () => {
```

Comment:
You mean check it's value is changing? I am looking into writing some more test cases for this. We need to probably send a message to the YouTube iframe to play so that it sends messages to our document that the player state has changed.